src/main/resources/i18n/messages.properties

```diff
@@ -990,6 +991,20 @@ project.settings.coverage.genome_size=Genome Size:
 project.settings.coverage.minimum_coverage=Minimum Coverage:
 project.settings.coverage.maximum_coverage=Maximum Coverage:
 
+
+# Project settings delete page
+DeleteProject.confirm=I understand that by deleting this project the above actions will occur
+DeleteProject.title=Delete Project
+DeleteProject.warning=Warning! Deletion of a project is permanent action!
+DeleteProject.consequences=Deleting a project will have the following consequences:
+DeleteProject.consequences.samples=Access to this project's samples will be removed for all users of this project.
+DeleteProject.consequences.copied=Samples which have been shared to other projects will not be affected.  They will still be available in their other projects.
+DeleteProject.consequences.reference=Reference files associated with this project will no longer be accessible.
+DeleteProject.consequences.associated=Associated projects will no longer have access to this project's samples.
+DeleteProject.consequences.analyses=Analyses shared with this project will no longer be shared with this project's users.  Submitted analyses will still be available to the submitting users in their analysis list.
```

Comment:
Change ending to `in their analyses list`

Reply:
Updated in 7eeb75b